### PR TITLE
Add RPCord wrapper (TypeScript)

### PR DESCRIPTION
**Pull request by DjDeveloperr**
I've made a [RPC wrapper](https://github.com/DjDeveloperr/RPCord) which supports all of documented features (and a bit more :p).

### CODE REVIEW

**Comment by crmarsh** [comment]
Aside: are people making games with node? A quick search didn't reveal any.

File: README.md [hunk 1]
```diff
@@ -155,4 +155,5 @@ Below is a table of unofficial, community-developed wrappers for and implementat
 | [drpc4k](https://github.com/Bluexin/drpc4k)                               | [Kotlin](https://kotlinlang.org/) |
 | [lua-discordRPC](https://github.com/pfirsich/lua-discordRPC)              | LuaJIT (FFI)                      |
 | [pypresence](https://github.com/qwertyquerty/pypresence)                  | [Python](https://python.org/)     |
+| [RPCord](https://github.com/DjDeveloperr/RPCord)                          | TypeScript                        |
```
Comment (Lachee) [inline README.md]: It should be after the Swift bindings. Alphabetical based on the language then library name.

**Review by Lachee — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (DjDeveloperr) [reply README.md]: Done 👍

**Review by CyberFlameGO — APPROVED** [review]

**Review by CyberFlameGO — APPROVED** [review]
LGTM

File: README.md [hunk 2]
```diff
@@ -156,3 +156,4 @@ Below is a table of unofficial, community-developed wrappers for and implementat
 | [lua-discordRPC](https://github.com/pfirsich/lua-discordRPC)              | LuaJIT (FFI)                      |
 | [pypresence](https://github.com/qwertyquerty/pypresence)                  | [Python](https://python.org/)     |
 | [SwordRPC](https://github.com/Azoy/SwordRPC)                              | [Swift](https://swift.org)        |
+| [RPCord](https://github.com/DjDeveloperr/RPCord)                          | TypeScript                        |
```
Comment (Panquesito7) [inline README.md:159]:
```suggestion
| [RPCord](https://github.com/DjDeveloperr/RPCord)                          | [TypeScript](https://www.typescriptlang.org/)                        |
```

**Pull request closed without merging**